packages/components/src/components/transfer-account-selector/transfer-account-selector.tsx

```diff
@@ -77,54 +65,17 @@ const TransferAccountSelector = ({
                 portalId={portal_id}
                 title={label}
                 toggleModal={() => setIsListModalOpen(old => !old)}
+                min_height={is_mobile ? mobile_list_min_height : ''}
```

Comment:
```suggestion
                min_height={is_mobile && mobile_list_min_height}
```

Reply:
@thisyahlen-deriv 
unfortunately here i can't do this, because TS complains that i cannot assign boolean to min_height